letters.js

```diff
@@ -0,0 +1,18 @@
+var Word = require("./Word");
+
+var Letter = function(letter) {
+	this.letter = letter;
+	this.alreadyGuessed = false;
+	this.show = false;
+	this.displayLetter = function() {
+		if (this.show == true) {
+			return this.letter;
+		}
+		else {
+			return "_ ";
+		}
+	}
+}
+
+
+module.exports = Letter;
```

Comment:
You aren't using Word anywhere so that can be taken out, also you could just write this like this:
```javascript
module.exports = function(letter) {
 	this.letter = letter;
 	this.alreadyGuessed = false;
 	this.show = false;
 	this.displayLetter = function() {
 		if (this.show == true) {
 			return this.letter;
 		}
 		else {
 			return "_ ";
 		}
	}
 }